unblob/models.py

```diff
@@ -116,6 +118,7 @@ def add_subtask(self, task: Task):
 
 @attr.define
 class ProcessResult:
+    # TODO: rethink the type of ProcessResult.results - is it a simple list?
```

Comment:
This TODO is not resolved in this PR. If we want to track this question, you should create an issue

Reply:
TODO was resolved (type has been changed).